GetPlayerArmour.md

```diff
@@ -0,0 +1,62 @@
+# SetPlayerHealth
```

Comment:
This file is in the root of the repository

Reply:
Moved